### ISSUE TITLE
Tell user why connection failed

### ISSUE THREAD

**Issue by kollix** [siacs/Conversations#1954]
#### General information

* **Version:**  1.13.2
* **Device:** Samsung Galaxy S4 mini
* **Android Version:** Android 4.4.2
* **Server name:**  jabber.at
* **Server software:** 
* **Installed server modules:** 
* **Conversations source:** F-Droid

#### Steps to reproduce

My daughter got a new phone and I assume she used a wrong password. Therefore the connection was not possible. Checking the problem, I saw that also my own phone could no longer connect (both phones were using the same WLAN router from my home network).
Checking with adb log I found out that the server blocked my (DSL) IP address due to multiple authentication failures.

However conversations just shows "Connecting..." and no hint whatsoever that it failed to connect and for what reason.

#### Expected result

I expect to get an information why the connection fails - at least in this case where the server very clearly states why it failes.

#### Debug output
````
07-13 19:08:23.640 D/conversations( 2390): martin.koller@jabber.at: connecting
07-13 19:08:23.640 D/conversations( 2390): using dns server: 192.168.129.3 to look up jabber.at
07-13 19:08:23.660 D/conversations( 2390): martin.koller@jabber.at: using values from dns xmpp.jabber.at[[2001:629:3200:95:0:0:1:40]]:5222 tls: false
07-13 19:08:23.660 D/conversations( 2390): martin.koller@jabber.at: failed to connect to /2001:629:3200:95::1:40 (port 5222) after 15000ms: connect failed: ENETUNREACH (Network is unreachable)(java.net.ConnectException)
07-13 19:08:23.660 D/conversations( 2390): martin.koller@jabber.at: using values from dns xmpp.jabber.at[128.130.95.40]:5222 tls: false
07-13 19:08:24.060 D/conversations( 2390): martin.koller@jabber.at: stream error <error xmlns="http://etherx.jabber.org/streams"><policy-violation xmlns="urn:ietf:params:xml:ns:xmpp-streams"/><text xmlns="urn:ietf:params:xml:ns:xmpp-streams" lang="en">Too many (5) failed authentications from this IP address (::FFFF:188.22.12.183). The address will be unblocked at 17:53:34 13.07.2016 UTC</text></error>
````

**Comment by licaon-kter** [comment]
Yeah, this could use some more info, I asked for more [somewhere else](https://github.com/siacs/Conversations/issues/1936#issuecomment-229585918), I'll quote: 

> 
> 
> Ok, so I though a bit more about this since I'm facing an issue, network gets cut off pretty aggressive in my current config.
> 
> Anyway, the options right now in Conversations are:
> 
>     account disabled -> small notification "account disabled: activate"
>     account enabled and connected+contact offline -> black send button
>     account enabled and connected+contact busy -> orange send button
>     account enabled and connected+contact connected -> green send button
> 
> But in my case, given the network/Conversation reconnection on screen on, I get another state:
> 
>     account enable BUT NOT connected -> black send button just like when actually connected (!?)
> 
> So I would like another small notification like when account is disabled, one that says something in the lines: account reconnecting in progress while it's reconnecting, security error, could not connect and what ever else is actually doing during a re-connect.

**Comment by keshto** [comment]
I agree. I would like to see disconnect and reconnects in the status bar like xabber has. Especially, when I am moving around it is nice to know if I am connected or not.

**Comment by iNPUTmice** [comment]
Should be fixed in 1.13.6
